subsys/bluetooth/host/hci_core.c

```diff
@@ -4521,3 +4454,45 @@ int bt_configure_data_path(uint8_t dir, uint8_t id, uint8_t vs_config_len,
 
 	return err;
 }
+
+/* Return `true` if a command was processed/sent */
+static bool process_pending_cmd(k_timeout_t timeout)
+{
+	if (!k_fifo_is_empty(&bt_dev.cmd_tx_queue)) {
+		if (k_sem_take(&bt_dev.ncmd_sem, K_NO_WAIT) == 0) {
+			hci_core_send_cmd();
+			return true;
+		}
+	}
+
+	return false;
+}
+
+#define _COMPILE_TX (IS_ENABLED(CONFIG_BT_CONN) ||		\
+		     IS_ENABLED(CONFIG_BT_ISO_UNICAST) ||	\
+		     IS_ENABLED(CONFIG_BT_ISO_BROADCASTER))
```

Comment:
Isn't this the same as `CONFIG_BT_CONN_TX`?

Reply:
it is, thanks